juju/client/facade.py

```diff
@@ -926,11 +926,22 @@ def generate_definitions(schemas):
     return definitions
 
 
-def generate_facades(schemas):
+def sortable_schema_version(version_string: str) -> Tuple[int, int, int]:
+    """Return a sorting key in the form (major, minor, micro) from a version string."""
+    # 'latest' is special cased in load_schemas and should come last
+    if version_string == 'latest':
+        return (9000, 9000, 9000)
```

Comment:
Why special-case latest?
We don't use files with the "latest" marker, do we?

Reply:
Late reply, but I special cased it in sorting because the code that loads in schemas special cases it as a permitted version. There was no need to special casing in sorting when sorting was simply lexicographic, since "latest" would sort after all the numerical strings.